Simulator/player.py

```diff
@@ -1868,6 +1894,22 @@ def won_round(self, damage):
                     fortune_returns[self.fortune_loss_streak])
                 self.fortune_loss_streak = 0
 
+    def add_augment(self, augment:Augment)-> bool:
```

Comment:
Match spacing and I've been trying to keep the methods in this file alphabetical. (Sae did a push a while back that kind of ruined some of it but his methods are related to the step function. This isn't a step function method so I would like it to be in order.)